src/repositories/firstLogin/firstLoginRepositoryImpl.ts

```diff
@@ -0,0 +1,30 @@
+import { GetSchoolListRespose } from "../../types/firstLogin/firstLogin.types";
+import { FirstLoginRepository, GetListParams, PostMajorParams, PostSchoolParams } from "./firstLoginRepository";
+import { dearV1Axios } from "@/libs/axios/customAxios";
+
+class FirstLoginRepositoryImpl implements FirstLoginRepository {
+  public async postSchool(params: PostSchoolParams): Promise<void> {
+    const { data } = await dearV1Axios.post("/school", params);
+    return data;
+  }
+
+  public async postMajor(params: PostMajorParams): Promise<void> {
+    const { data } = await dearV1Axios.post("/school/major", params);
+    return data;
+  }
+
+  public async getSchoolList(params: GetListParams): Promise<GetSchoolListRespose> {
+    const { gubunType, keyword } = params;
+    const { data } = await dearV1Axios.get(`/school/list?gubunType=${gubunType}&keyword=${keyword}`);
+    return data;
+  }
+
+  public async getMajorList(params: GetListParams): Promise<GetSchoolListRespose> {
+    const { gubunType, keyword } = params;
+    const { data } = await dearV1Axios.get(`/school/major/list?gubunType=${gubunType}&keyword=${keyword}`);
+    return data;
+  }
+}
+
+const firstLoginRepositoryImpl = new FirstLoginRepositoryImpl();
+export default firstLoginRepositoryImpl;
```

Comment:
필요없는 변수 선언 인거 같네요
default 할때 바로 new FirstLoginRepositoryImpl  하는거 더좋을거 같습니다 
```suggestion
export default new FirstLoginRepositoryImpl();
```

Reply:
넵 class쓸때 습관이 들려있었던것 같습니다 조심하겠습니다.
수정하였습니다